server/datastore/mysql/labels.go

```diff
@@ -547,10 +549,6 @@ func (ds *Datastore) applyHostLabelFilters(filter fleet.TeamFilter, lid uint, qu
 		query += ` JOIN host_display_names hdn ON h.id = hdn.host_id `
 	}
 
-	if opt.MDMIDFilter != nil || opt.MDMEnrollmentStatusFilter != "" {
-		query += ` JOIN host_mdm hmdm ON h.id = hmdm.host_id `
-	}
```

Comment:
Just to double-check, this isn't required/supported anymore?

Reply:
The filters are still supported, but the JOIN will always be included in the base query.